README.md

```diff
@@ -11,7 +11,7 @@
 
 ## Live
 
-here (https://eroiyo.github.io/To-do-list/)
+here ()
```

Comment:
- [ ] Remove the live link link section if you don't have. But I would recommend to add live link by using [github pages](https://guides.github.com/features/pages/).